### DISCUSSION TITLE
Customizing skin with graphs.conf? CSS?

### DISCUSSION THREAD

**Discussion by damclarkd** [poblabs/weewx-belchertown#782] [Q&A] [answered]
After many months of running the Belchertown skin "Out of the box" configuration I am starting to add more graphs to the graph sections and the front-page using graphs.conf. Using the excellent Belchertown Charts wiki, I have some new charts on my front page.
My question might be more of a css question than a chart question, but stuck on how to proceed. For example I have added a YTD weatherRadial graph and would like it to span the page width not just the column width. Adding width = 100% in the graph.conf did not change the display. (To be fair the wiki said as much, but I tried anyway.)
I notice in the wiki example there is a css_class = "weatherRadial" option. Would I somehow incorporate that in a custom.css file?
Any pointers in the right direction to get this started are appreciated.
Thank you.
Cheers!

**Comment by ThePeePs** [comment]
Adding width = 100% should make the graph be the full width of the page which is set in css as 1200px.  If you want your page to be wider, you'd need to change the "max-width" setting for the `site-inner` class.  This can be done by creating a css file named "custom.css" in the root of the skin directory.

**Reply by damclarkd** [reply]
So I have the width = 100% in graph.conf for that particular graph. What I am trying to do is after the 1st four default graphs on the home page, which are in two columns of two, is add a new graph that spans both columns. The site does not need to be wider. Just the one graph. 
I guess the issue might be the two columns defined for the graph area on the home page?

**Comment by ThePeePs** [comment]
If you look at [my site](https://weather.thepeeps.net) and scroll down you'll see that I have the year's temp range the full width of the site. 
Is that what you're looking to do?
Here is snippet of my `graphs.conf`

```
    [[TemperatureRange]]
        title = Temperature Range This Year
        show_button = true
        button_text = Weather Radial
        time_length = year
        width = "100%"
        height = "500px"
        css_class = "weatherRadial"
        [[[weatherRange]]]
            range_type = outTemp
            area_display = 1
```

**Reply by damclarkd** [reply]
Yes. That is the goal. From my graph.conf...
[[chart5]]
        title = Temperature Ranges This Year
        css_class = "weatherRadial"
        [[[weatherRange]]]
            range_type = outTemp
            time_length = year
            width = 100%

**Comment by ThePeePs** [comment, marked as answer]
I think you need to move the `width = 100%` to above the `[[[weatherRange]]]`, also I think the `100%` needs to be in quotes.

**Reply by damclarkd** [reply]
That solved it for me! Following your example I moved and added some other elements also. Working how I expected now
Thanks very much for your assistance. It is appreciated.
Cheers!

**Comment by damclarkd** [comment]
I’ll give that a try. Thanks. 

Dennis
Follow along at www.damclark.com
